tools/compogen/pkg/gen/resources/templates/readme.mdx.tmpl

```diff
@@ -42,11 +42,22 @@ ${connection.<my-connection-id>}`.
 
 | Field | Field ID | Type | Note |
 | :--- | :--- | :--- | :--- |{{ range . }}
-| {{ .Title }}{{ if .Required }} (required){{ end }} | `{{ .ID }}` | {{ .Type }} | {{ .Description }} |{{ end }}
+| {{ .Title }}{{ if .Required }} (required){{ end }} | `{{ .ID }}` | {{ .Type }} | {{ .Description }} {{ if .Enum }} There are options: <br/>- {{ enumValues .Enum }}  {{ end }} |{{ end }}
```

Comment:
`There are options` feels a bit awkward. I'd add a line break, then the list of enums:
```
The name of the model used for tokenization.
Enum:
- `gpt-4`
- `gpt-3.5-turbo`
- ...
```

I'd also add the enums as monospaced strings and, in order to properly render the bullet list, use `<ul>` and `<li>` HTML tags ([ref](https://stackoverflow.com/questions/19950648/how-to-write-lists-inside-a-markdown-table))